delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
+
+    $query = "DELETE FROM books WHERE id = $id";
+    if (mysqli_query($connection, $query)) {
+        echo "Book deleted successfully!";
+        header("Location: list_books.php");
+    } else {
+        echo "Error: " . mysqli_error($connection);
+    }
+}
+?>
```

Comment:
Consider using prepared statements to prevent SQL injection, and verify input ID is a valid integer.